persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbPatientRepository.java

```diff
@@ -28,4 +28,15 @@ MetadbPatient findPatientByCmoPatientId(
             + "MATCH (s)<-[:HAS_SAMPLE]-(p: Patient) "
             + "RETURN p.metaDbPatientId")
     UUID findPatientIdBySample(@Param("metaDbSampleId") UUID metaDbSampleId);
+
+    //needs to be tested with clinical samples
+    @Query("MATCH (s: Sample)<-[:IS_ALIAS]-(sa: SampleAlias{value: $dmpId, namespace: 'dmpId'}) "
+            + "MATCH (s)<-[:HAS_SAMPLE]-(p: Patient) "
+            + "RETURN p")
+    MetadbPatient findPatientByClinicalSample(@Param("dmpId") String dmpId);
```

Comment:
I'm thinking that these methods can be replaced by a query that fetches by a sample alias namespace + value
```
MetadbPatient findPatientBySampleAlias(@Param("namespace") String namespace, @Param("value") String value);
```